### ISSUE TITLE
Filter tags at the sidebar is not working

### ISSUE THREAD

**Issue by icahlatif** [topcoder-platform/community-app#3734]
Target URL: https://www.topcoder.com/thrive/search?tags[0]=Topcoder-centric&tags[1]=Tips&author=All%20authors&startDate=2020-01-07T10%3A54%3A25%2B08%3A00&endDate=2020-02-07T10%3A54%3A25%2B08%3A00&track=&tax=

**Reproduction Steps**
1. Go to https://www.topcoder.com/thrive/
2. Scroll down to FEATURED ARTICLES section
3. Click on the "Topcoder-centric" tag 
4. Type "Tips" at the Add tags.. text box
5. Click on the APPLY FILTER button

**Expected Result**
The Search Results should show results of articles with the similar tags

**Actual Result**
The Search Results show "Nothing Found"

**Screenshots**
![tags](https://user-images.githubusercontent.com/57003591/73998400-7aab6480-499c-11ea-949e-c6098fd3be3a.png)

**Screencast**
[filter tags.zip](https://github.com/topcoder-platform/community-app/files/4168846/filter.tags.zip)

**OS/Browser Information**
 - OS: Windows 10 Home Single Language 64-bit
 - Browser Google Chrome
 - Version 79.0.3945.130 

**Bug Label** <Functional>

**Comment by icahlatif** [comment]
Bug Label : Functional

**Comment by winterflame** [comment]
![article](https://user-images.githubusercontent.com/4590317/74323480-8776f080-4dc0-11ea-9cd0-281774560c68.jpg)

There's one article with tags "Tips" and "Topcoder-centric".

**Comment by icahlatif** [comment]
Yes, but that only happened when clicking the tag button from the https://www.topcoder.com/thrive/ not using the "Apply Filter" button
